test/fixtures/track-basic.json

```diff
@@ -32,7 +32,7 @@
       "$browser": "Safari",
       "$os": "iPhone OS",
       "$os_version": "8.1.3",
-      "$browser_version": "7.1.0.7",
+      "$browser_version": 7.1,
```

Comment:
mixpanel seems to allow it to be a string or a number. the client side library sends it as a number, so I switched this to do the same.